src/youwol/app/environment/models/models_config.py

```diff
@@ -1289,9 +1290,13 @@ async def is_matching(self, incoming_request: Request, context: Context):
     async def switch(
         self, incoming_request: Request, context: Context
     ) -> Optional[Response]:
+        # The following 'Any' is because the implementation of this class is not separated from its declaration as
```

Comment:
A better solution might be to put  FlowSwitch classes’ hierarchy into its own module:
* it will avoid circular dependency
* it will reduce this bloated module

Reply:
Let's find out the simplest solution, I have created an issue to tackle this specifically (TG-1996). I would like to do it in a separate PR.

Reply:
The modules reorganization is part of the PR.